common/app/views/fragments/atoms/chart.scala.html

```diff
@@ -0,0 +1,7 @@
+@import com.gu.contentatom.renderer.{ArticleAtomRenderer, ArticleConfiguration}
+
+@(chart: model.content.ChartAtom, config: ArticleConfiguration)
+
+<div class="u-responsive-ratio">
+    @Html(ArticleAtomRenderer.getHTML(chart.atom, config))
```

Comment:
Oh, I see. Great 👍

Reply:
Yeah, I'm trying to get the iframe's body to be the correct height. Wrapping it in `u-responsive-ratio` gets somewhere close, but results in a lot of whitespace (see screenshot).

Reply:
It is unfortunately really hard to do. The next solution would be to use web components 😎